### PR TITLE
Update upgrade state values in ocm-sdk

### PR DESCRIPTION
**Pull request by zvikorn**
Signed-off-by: Tzvika Avni <tavni@redhat.com>

https://issues.redhat.com/browse/SDA-3662

### CODE REVIEW

**Comment by zvikorn** [comment]
@igoihman @vkareh @jhernand @tzvatot PTAL

**Comment by vkareh** [comment]
LGTM

File: model/clusters_mgmt/v1/UpgradePolicyStateValue.model [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Overall state of a cluster upgrade policy.
+enum UpgradePolicyStateValue {
+    // Upgrade policy set but an upgrade wasn't scheduled yet
+    Pending
+
+    // Upgrade policy set an an upgrade was scheduled
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/UpgradePolicyStateValue.model:22]: `an an` -> `and`

File: model/clusters_mgmt/v1/UpgradePolicyStateValue.model [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Failed
+
+    // Upgrade is taking longer than expected
+    Delayed
+
+    // Upgrade got cancelled (temporary state - the policy will get removed).
+    Cancelled
+
+    // The state of the upgrade policy is unknown.
+    Unknown
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/UpgradePolicyStateValue.model:42]: There's no such state

File: model/clusters_mgmt/v1/upgrade_policy_state_type.model [hunk 3]
```diff
@@ -16,8 +16,9 @@ limitations under the License.
 
 // Representation of an upgrade policy state that that is set for a cluster.
 class UpgradePolicyState {
-	// State value, can be 'pending', 'started', 'delayed', 'failed' or 'completed'.
-	Value String
+	// State value, can be 'pending', 'scheduled', 'canceled', 'started', 'delayed',
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/upgrade_policy_state_type.model:19]: NP: redundant comma

File: model/clusters_mgmt/v1/upgrade_policy_state_type.model [hunk 4]
```diff
@@ -16,8 +16,9 @@ limitations under the License.
 
 // Representation of an upgrade policy state that that is set for a cluster.
 class UpgradePolicyState {
-	// State value, can be 'pending', 'started', 'delayed', 'failed' or 'completed'.
-	Value String
+	// State value can be 'pending', 'scheduled', 'canceled', 'started', 'delayed',
```
Comment (vkareh) [inline model/clusters_mgmt/v1/upgrade_policy_state_type.model:19]: In the enum above you have 'cancelled', but here it says 'canceled' - I think we should use the 'cancelled' spelling variant, as it applies more internationally

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**